pkg/tfgen/generate.go

```diff
@@ -1800,3 +1803,20 @@ func ignoreMappingError(s []string, str string) bool {
 	}
 	return false
 }
+
+func addRenamesToMetadataInfo(g *Generator) error {
```

Comment:
This has the side-effect of adding renames to providers that don't strictly need them, but opt into MetadataInfo.